purchases/src/test/java/com/revenuecat/purchases/common/verification/SigningManagerTest.kt

```diff
@@ -191,6 +191,22 @@ class SigningManagerTest {
         assertThat(verificationResult).isEqualTo(VerificationResult.VERIFIED)
     }
 
+    @Test
+    fun `verifyResponse with encoded url verifies correctly`() {
+        val rootVerifier = DefaultSignatureVerifier("yg2wZGAr8Af+Unt9RImQDbL7qA81txk+ga0I+ylmcyo=")
+        val signingManager = SigningManager(
+            SignatureVerificationMode.Informational(IntermediateSignatureHelper(rootVerifier)),
+            appConfig,
+            apiKey,
+        )
+        val verificationResult = callVerifyResponse(
+            signingManager,
+            requestPath = "/v1/subscribers/\$RCAnonymousID%3A1af512a3b9c848899fe427f39dd69f2b",
+            signature = "xoDYyUeHnIlSIAeOOzmvdNPOlbNSKK+xE0fE/ufS1fsAAMNQ1HiPDL34Vx0Uy74KPV5mztuk3DHBpucT/rSYVlkxIa3ModYmPfYZ20lnlbSB1UiP6oJHwAA2pXlS6AQ5eLSuAmm2UIYPrDGEEC8Lgj1sAn2fGMRMx2eaPNzDPBGTxxZROfjkI1wtsyJC0w0I7d8TkLeXjUTlWNafmc4GVMleE/tQZZGIoNrnar0HqICUnB8B",
```

Comment:
just double-checking, these were obtained with a signature that isn't set to expire, right? Otherwise, are we ensuring that these won't go stale when the intermediate key goes stale?

Reply:
I generated these with a date very far into the future (Year 2106) so they shouldn't expire. Since we aren't using real keys for these tests, it should be ok.